### PR TITLE
Gradeuser and monitor beautification

### PR DESCRIPTION
**Pull request by maheshgudi**
1. Gradeuser and monitor answerpapers are beautified.
2. Monitor table can be sorted according to columns (marks, name etc).

screenshot from 2016-09-15 16 10 38
screenshot from 2016-09-15 16 11 58

### CODE REVIEW

File: yaksh/static/yaksh/css/view_answerpaper.css [hunk 1]
```diff
@@ -58,4 +58,19 @@ code{
 }
 pre{
   background-color: transparent;
+}
+
+/*for grade user*/
+textarea
+{
+width : 100%;
```
Comment (prabhuramachandran) [inline yaksh/static/yaksh/css/view_answerpaper.css]: Please keep the indentation consistent.

File: yaksh/static/yaksh/css/view_answerpaper.css [hunk 2]
```diff
@@ -58,4 +58,19 @@ code{
...
+height : 15%;
+}
+#attempt {
+    width: 157px;
+    position: relative; left:20%;
+}
+#myContent {
+margin: 5px 10px;
```
Comment (prabhuramachandran) [inline yaksh/static/yaksh/css/view_answerpaper.css]: Same here: indentation.

File: yaksh/templates/yaksh/grade_user.html [hunk 3]
```diff
@@ -5,11 +5,7 @@
 {% block subtitle %} Grade User {% endblock %}
  
 {% block css %}
-     <link rel="stylesheet" href="{{ URL_ROOT }}/static/yaksh/css/gradeuser.css" type="text/css" /> 
-{% endblock %}
-
-{% block script %}
-    <script src= "{{ URL_ROOT }}/static/yaksh/js/edit_question.js"></script>
+     <link rel="stylesheet" href="{{ URL_ROOT }}/static/yaksh/css/view_answerpaper.css" type="text/css" />
```
Comment (prabhuramachandran) [inline yaksh/templates/yaksh/grade_user.html]: Wrap lines to 80 cols when possible.

File: yaksh/templates/yaksh/grade_user.html [hunk 4]
```diff
@@ -125,42 +122,63 @@
...
+    Marks: <input id="q{{ question.id }}" type="text" 
         name="q{{ question.id }}_marks" size="4"
         value="{{ answer.marks }}"><br><br>
-{% endwith %}
-</div>
-</div>
-{% endfor %} {# for question, answers ... #}
-
+    {% endwith %}
+    <hr>
```
Comment (prabhuramachandran) [inline yaksh/templates/yaksh/grade_user.html]: Should this be `<hr/>`?

File: yaksh/static/yaksh/css/view_answerpaper.css (hunk 1 above)
Comment (maheshgudi) [inline yaksh/static/yaksh/css/view_answerpaper.css]: resolved.

File: yaksh/static/yaksh/css/view_answerpaper.css (hunk 2 above)
Comment (maheshgudi) [inline yaksh/static/yaksh/css/view_answerpaper.css]: resolved.

File: yaksh/templates/yaksh/grade_user.html (hunk 4 above)
Comment (maheshgudi) [inline yaksh/templates/yaksh/grade_user.html]: resolved.

**Comment by maheshgudi** [comment]
Resolved all issues.

**Pull request merged**